tests/_support/index.js

```diff
@@ -600,6 +600,28 @@ Cypress.Commands.add("configureCluster", function(configuration) {
       });
   }
 
+  if (configuration.communityPackages) {
```

Comment:
A new configuration I added. It's very similar to the existing one, expect it's a community package.